vulnerabilities/importers/example.py

```diff
@@ -0,0 +1,97 @@
+from datetime import datetime
```

Comment:
It makes sense to have it as a test... but since this is a test, why not move `
vulnerabilities/importers/example.py` to the tests directory instead? This would make more sense.
Especially I would like to move the tests out of the main code directories at some point in time in the future

Reply:
The initial idea behind putting it in ``vulnerabilities/importers/`` was to offer a starting point to contributors about how a real importer/improver might look like but that is covered in detail in the documentation as well so we might as well move these to ``tests``